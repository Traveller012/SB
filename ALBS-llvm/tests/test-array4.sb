{ : int} main = [
	
	chr[] a;
	int i;
	a = new chr[5]; 

	a[0] = 'b';
	a[0] = 'a';
	
	print(a[0]);

	a[1] = 'b';
	a[2] = 'c';
	a[3] = 'd';
	a[4] = 'e';

	i = 0;
	while (i < 5) {
		print(a[i]);
		i = i + 1;
	}
	  
	rtn 0;
]
