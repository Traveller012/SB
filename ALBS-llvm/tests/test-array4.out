a
a
b
c
d
e